tools/wakucanary/README.md

```diff
@@ -45,3 +48,9 @@ $ ./build/wakucanary --address=/dns4/node-01.do-ams3.status.test.statusim.net/tc
 $ echo $?
 0
 ```
+
+Websockets are also supported.
```

Comment:
Since the websockets port is not configurable (and probably doesn't need to be), we may want to just document here where the websockets port will be opened (in this case `node-port + 1000`).

Reply:
> LGTM, thanks for this! May want to ping @felicio to hear if they need any canary service deployment to https://canary.infra.status.im/ to monitor websockets ports on the fleet. I'd also like to get @alrevuelta's review before we merge as this is his tool, but good job. :)

> Nice. And yes please, both prod and test incl. relay, lightpush, filter and store.
 –  https://discord.com/channels/864066763682218004/1093501948105338962/1093563071622029362

Reply:
:+1: